src/Acquisition/tests.py

```diff
@@ -3278,7 +3309,7 @@ class B(Implicit):
         base = B()
         base.value = 42
 
-        rich_cmp_methods = ['__lt__', '__gt__', '__eq__',
+        rich_cmp_methods = ['__lt__', '__gt__',  '__eq__',
```

Comment:
This change does not seem to be necessary.

Reply:
Redundant blank removed.